radio/src/storage/conversions/yaml/dataconstants_220.h

```diff
@@ -58,7 +58,7 @@
   // #define NUM_SLIDERS                   0
   #define STORAGE_NUM_POTS              1
   #define STORAGE_NUM_SLIDERS           0
-#elif defined(RADIO_T8) || defined(RADIO_TLITE)
+#elif defined(RADIO_T8) || defined(RADIO_TLITE) || defined(RADIO_COMMANDO8)
```

Comment:
Please remove all changes in `storage/conversions/*`: you would only need them to support a pre-existing binary storage. As EdgeTx does not support binary storage for new radios, it should not be needed.